Gopkg.lock

```diff
@@ -549,6 +549,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "01f42491c7ac8bb0d47771086f629f63dc66bbaa9ce9a595b681d3e362e4bbcc"
+  inputs-digest = "309fa1a361fc845e248eb30c6e4c163b9c19ac974b011a432e14374f05366597"
```

Comment:
Unrelated change?

Reply:
Maybe, but if so we need to fix something in make ... Will take a look in the morning. 👍 🦁🦁🦁

Reply:
running `make` on a fresh master causes an update to the lock ... we must have missed it on a differnt PR ... will remove from this and submit a PR to do this update 👍